source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/CalculationResults/Statements/RequestSpecificWholesaleServicesQuerySnippetsProviderFactory.cs

```diff
@@ -16,16 +16,16 @@
 
 namespace Energinet.DataHub.Wholesale.CalculationResults.Infrastructure.CalculationResults.Statements;
 
-public class WholesaleServicesQueryStatementHelperFactory(
-    IEnumerable<IWholesaleServicesDatabricksContract> helperComponents)
+public class RequestSpecificWholesaleServicesQuerySnippetsProviderFactory(
```

Comment:
```suggestion
public class RequestSpecificWholesaleServicesQuerySnippetsProviderFactory(
```

What is the difference between a 'specific" and 'non-specific' wholesaleservice?